src/components/sidebar/VoyagePageButton.tsx

```diff
@@ -43,7 +43,7 @@ export default function VoyagePageButton({
         <button
           type="button"
           className={`bg-transparent hover:bg-transparent w-[9.375rem] h-[1.1875rem] min-h-0 mb-2.5 flex justify-start ${
-            isVoyageStarted ? "pl-11" : "pl-6 pointer-events-none"
+            isVoyageStarted ? "pl-11" : "pl-6"
```

Comment:
This is good but add the pointer-events-none to the link above this with the same conditional. If the voyage is not started, this class should be applied.

Reply:
@Dan-Y-Ko Sorry but I do not understand, you mean something like that?

```
      <Link
        href={isVoyageStarted ? element.link : "#"}
        className={!isVoyageStarted ? "pointer-events-none" : ""}
      >
```

Reply:
yea exactly but I think this would be better 

```
<Link
        href={isVoyageStarted ? element.link : "#"}
        className={isVoyageStarted ? "" : "pointer-events-none"}
>
```